### DISCUSSION TITLE
World of Light Crashes on Corps battles

### DISCUSSION THREAD

**Discussion by InAudibleDayne** [Raytwo/ARCropolis#579] [Issues]
After updating Arcropolis to 4.0.2 and using the latest switch (17.0.1) and smash (13.0.2) versions Corps battles consistently crash when trying to load the second fighter spawn. (if there are 4 enemies and I kill all 4, the 5th fighter will load but all fighters afterwards fail to load. 6th fighter doesn't spawn and when the 7th is queued the game crashes). I have tested with all mods disabled and only using Arcropolis and still have the issue.

**Comment by InAudibleDayne** [comment]
Sorry if this has been reported before I searched key words like World of Light and Adventure Mode but didn't see any mention of it.

**Comment by InAudibleDayne** [comment]
Doesn't seem to have a problem loading multiple Snake fighters. But does seem to crash anytime more than 4 Game and Watches are loaded in one of the spirit battles.

**Comment by rafaelkai** [comment]
This happens just for Mr Game and Watch corps fights for me. The rest seem to work fine. It's specially annoying since most of Mr Game and Watch WOL fights are corps though.
